shell/platform/embedder/embedder.cc

```diff
@@ -45,6 +45,8 @@ extern const intptr_t kPlatformStrongDillSize;
 const int32_t kFlutterSemanticsNodeIdBatchEnd = -1;
 const int32_t kFlutterSemanticsCustomActionIdBatchEnd = -1;
 
+#define GL_TEXTURE_RECTANGLE 0x84F5
```

Comment:
Use static constants, not #define, for constant values; macros are very strongly discouraged by the style guide.

Also, please add a comment explaining this magic value.